### ISSUE TITLE
Oauth2 on localhost

### ISSUE THREAD

**Issue by globegithub** [thechutrain/mern-passport#5]
I'd appreciate it if you could list DETAILED steps for testing Google Oauth2 while deployed on localhost for a newbie.

**Comment by Ignacio1996** [comment]
Yes please, I've done passport in the past and cannot figure this out.
